src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/licence/Licence.kt

```diff
@@ -22,36 +26,43 @@ data class Licence(
   )
   val version: String,
 
-  @Schema(description = "The current status code for the licence", example = "IN_PROGRESS")
+  @Schema(
+    description = "The current status code for the licence. This is non exhaustive and subject to change",
+    example = "IN_PROGRESS",
+    allowableValues = [
+      "IN_PROGRESS", "SUBMITTED", "APPROVED", "ACTIVE", "VARIATION_IN_PROGRESS",
+      "VARIATION_SUBMITTED", "VARIATION_APPROVED",
+    ],
```

Comment:
I think this is prepopulated by the LicenceStatus enum? Probably better not to have to keep this list up to date?

Reply:
the benefit of having our own licence status enum is that we can fix it to only have the allowed values

Reply:
The current LicenceStatus enum contains statuses which we discussed might not be needed hence this version of it (which is missing VARIATION_REJECTED). Note below from [here](https://docs.google.com/document/d/1CrlAGx19jT43Uia1a_0CWPz9BG-CnkaoX4VGn5LRHC4/edit):

- statusCode - this is non exhaustive and subject to change
- rejected does not exist but VARIATION_REJECTED does
- remove INACTIVE

Reply:
Ignore me - just found the Public API specific LicenceStatus enum